tests/etl/test_perf_data_adapters.py

```diff
@@ -50,4 +51,22 @@ def test_adapt_and_load():
         datum['blob'] = json.dumps({'talos_data': [datum['blob']]})
         tda.adapt_and_load(reference_data, job_data, datum)
 
+        # we upload a summary with a suite and subtest values, +1 for suite
+        if 'summary' in datum['blob']:
```

Comment:
in this case I have modified some of the data in the sample data so we can
test the original format and the new format.

On Wed, Aug 5, 2015 at 4:46 PM, William Lachance <notifications@github.com>
wrote:

> In tests/etl/test_perf_data_adapters.py
> <https://github.com/mozilla/treeherder/pull/836#discussion_r36352056>:
>
> > @@ -50,4 +51,22 @@ def test_adapt_and_load():
> >          datum['blob'] = json.dumps({'talos_data': [datum['blob']]})
> >          tda.adapt_and_load(reference_data, job_data, datum)
> >
> > +        # we upload a summary with a suite and subtest values, +1 for suite
> > +        if 'summary' in datum['blob']:
>
> I think this condition will always be true with your patch? Ideally we
> want to test both codepaths: one where the talos blob has a summary, one
> where it doesn't/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/treeherder/pull/836/files#r36352056>.
>